core/DataTable.php

```diff
@@ -1292,7 +1293,7 @@ public function getSerialized($maximumRowsInDataTable = null,
         // we then serialize the rows and store them in the serialized dataTable
         $rows = array();
         foreach ($this->rows as $id => $row) {
-            if (array_key_exists($id, $consecutiveSubtableIds)) {
+            if (isset($consecutiveSubtableIds[$id])) {
```

Comment:
It is known that array_key_exists is quite a bit slower than `isset`. As this is executed easily a few million times this makes a difference in performance but here `array_key_exists` and `isset` work the same in logic as we never set `$consecutiveSubtableIds[$id]=null`